docs/source/format/CanonicalExtensions.rst

```diff
@@ -283,6 +283,155 @@ UUID
    A specific UUID version is not required or guaranteed. This extension represents
    UUIDs as FixedSizeBinary(16) with big-endian notation and does not interpret the bytes in any way.
 
+Unknown
+=======
+
+Unknown represents a type or array that an Arrow-based system received from an
+external (often non-Arrow) system, which it cannot interpret itself or did not
+have support for in advance.  In this case, it can pass on Unknown to its own
+clients to communicate that a field exists, but that it cannot interpret the
+field or data.
+
+Extension parameters:
+
+* Extension name: ``arrow.unknown``.
+
+* The storage type of this extension is any type.  If there is no underlying
+  data, the storage type should be Null.  If there is data, the storage type
+  should preferably be binary or fixed-size binary, but may be any type.
```

Comment:
why do we say that it should preferably be binary or fixed-size binary?

Reply:
Originally @pitrou's suggestion IIRC, but most likely if you have uninterpretable data it's probably a binary blob.

Reply:
ok, I understand that it will probably be a binary blob but saying on the spec that `should probably be` was confusing to me. Thanks for the clarification

Reply:
I removed it, it's not so necessary (and it leaves out BinaryView anyways)